src/pages/workspace/withPolicy.tsx

```diff
@@ -99,9 +93,9 @@ const policyDefaultProps: WithPolicyOnyxProps = {
  */
 export default function <TProps extends WithPolicyProps, TRef>(WrappedComponent: ComponentType<TProps & RefAttributes<TRef>>): React.ComponentType<Omit<TProps, keyof WithPolicyOnyxProps>> {
     function WithPolicy(props: TProps, ref: ForwardedRef<TRef>) {
-        const routes = useNavigationState((state) => state.routes || []);
-        const currentRoute = routes?.at(-1);
-        const policyID = getPolicyIDFromRoute(currentRoute as PolicyRoute);
+        const routes = useNavigationState((state: NavigationState<Record<string, {policyID?: string}>>) => state.routes);
```

Comment:
and also this `Record<string, {policyID?: string}>` can be move to seperate type